nbresuse/metrics.py

```diff
@@ -0,0 +1,47 @@
+from typing import NamedTuple
```

Comment:
The motivation was that I felt if we wanted to add more metrics, it would be easier to keep the metric calculation and metric publishing separate. In that case if you wanted to change for example how the metrics are published, it would be a easier diff. For example if you wanted to keep that legacy endpoint.

Reply:
OK, that makes sense to me.